assets/src/edit-story/components/form/advancedDropDown/container.js

```diff
@@ -41,6 +41,15 @@ const Container = styled.div`
   border-radius: ${({ theme }) => theme.border.radius.default};
   padding: 5px;
   margin-top: 16px;
+  ${({ displayInContent }) =>
```

Comment:
Hm, not sure I understand the naming of this flag. First off, I prefer naming boolean flags with a `is`, `has` or `should` prefix to distinguish them.

But more importantly, isn't this doing the opposite of what it's called? Could we name it `isInline` and if true, use `position:relative`, if false `position:absolute`? Or something like that?

Currently the name sounds like it's supposed to be inline if true, but is not inlined, rather it's absolute

EDIT: See comment below about my confusion.